multibls/multibls.go

```diff
@@ -13,7 +15,8 @@ type PrivateKey struct {
 
 // PublicKey stores the bls public keys that belongs to the node
 type PublicKey struct {
-	PublicKey []*bls.PublicKey
+	PublicKey      []*bls.PublicKey
+	PublicKeyBytes []shard.BLSPublicKey
```

Comment:
I see why we are using Bytes in naming here. Can we rename PublicKey as `libPubKey` or something else?

Reply:
Will do more refactoring in next PR.